platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -99,41 +99,46 @@ - (instancetype)initWithOptions:(MGLMapSnapshotOptions *)options
         _options = options;
         _loading = false;
         
-        mbgl::DefaultFileSource *mbglFileSource = [MGLOfflineStorage sharedOfflineStorage].mbglFileSource;
-        _mbglThreadPool = mbgl::sharedThreadPool();
-        
-        std::string styleURL = std::string([options.styleURL.absoluteString UTF8String]);
-        
-        // Size; taking into account the minimum texture size for OpenGL ES
-        // For non retina screens the ratio is 1:1 MGLSnapshotterMinimumPixelSize
-        mbgl::Size size = {
-            static_cast<uint32_t>(MAX(options.size.width, MGLSnapshotterMinimumPixelSize)),
-            static_cast<uint32_t>(MAX(options.size.height, MGLSnapshotterMinimumPixelSize))
-        };
-        
-        float pixelRatio = MAX(options.scale, 1);
-        
-        // Camera options
-        mbgl::CameraOptions cameraOptions;
-        if (CLLocationCoordinate2DIsValid(options.camera.centerCoordinate)) {
-            cameraOptions.center = MGLLatLngFromLocationCoordinate2D(options.camera.centerCoordinate);
-        }
-        cameraOptions.angle = MAX(0, options.camera.heading) * mbgl::util::DEG2RAD;
-        cameraOptions.zoom = MAX(0, options.zoomLevel);
-        cameraOptions.pitch = MAX(0, options.camera.pitch);
-        
-        // Region
-        mbgl::optional<mbgl::LatLngBounds> coordinateBounds;
-        if (!MGLCoordinateBoundsIsEmpty(options.coordinateBounds)) {
-            coordinateBounds = MGLLatLngBoundsFromCoordinateBounds(options.coordinateBounds);
-        }
-        
-        // Create the snapshotter
-        _mbglMapSnapshotter = std::make_unique<mbgl::MapSnapshotter>(*mbglFileSource, *_mbglThreadPool, styleURL, size, pixelRatio, cameraOptions, coordinateBounds);
+        [self configureMBGLMapSnapshotter:options];
```

Comment:
Both call sites to `-configureMBGLMapSnapshotter:` pass in the value of the `options` property. Since it isn’t necessary to support configuring the receiver based on an arbitrary options object, let’s remove this parameter and have `-configureMBGLMapSnapshotter` refer to the `options` property.

The usual name for a method like this is `-commonInit`, as seen in MGLMapView.